h1st/model/fuzzy/fuzzy_logic_model.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from typing import Dict
+
+from h1st.model.rule_based_model import RuleBasedModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyLogicModel(RuleBasedModel):
+    """
+    This FuzzyLogicModel is one kind of rule-based model. You can encode your
+    rules in a fuzzy way using variables with membership functions.
+    Add variables and use those variables when defining rules.
+
+    """
+    def __init__(self):
+        super().__init__()
+
+    def execute_rules(self, input_data: Dict) -> Dict:
+        if self.rules is None:
```

Comment:
Where did `self.rules` come from? Make sure this is clear within this file/class.